DESCRIPTION

```diff
@@ -20,13 +20,16 @@ Imports:
     miniUI (>= 0.1.1),
     shiny (>= 0.13.0),
     shinyjs (>= 0.5.2),
-    utils
+    utils,
+    grDevices
 Suggests:
     knitr (>= 1.7),
     rmarkdown,
-    rstudioapi (>= 0.5)
+    rstudioapi (>= 0.5),
+    testthat,
```

Comment:
Sure...On a related subject, have you seen this? http://dirk.eddelbuettel.com/blog/2017/03/22/